app/lib/actions/remote_execution/run_host_job.rb

```diff
@@ -37,6 +37,10 @@ def humanized_output
         return output.join("\n")
       end
 
+      def humanized_name
+        _('Run ') + input[:job_name] + _(' on ') + input[:host][:name]
```

Comment:
`_('Run %{job_name} on %{host}") % { :job_name => input[:job_name], :host => input[:host][:name] }` is better format for translators, as for example translation for 'on' might depend on context, as well as the word ordering